UefiPayloadPkg/UniversalPayloadBuild.py

```diff
@@ -321,6 +321,7 @@ def main():
     parser.add_argument("-af", "--AddFv", type=ValidateAddFv, action='append', help='Add or replace specific FV into payload, Ex: uefi_fv=XXX.fv')
     parser.add_argument("-f", "--Fit", action='store_true', help='Build UniversalPayload file as UniversalPayload.fit', default=False)
     parser.add_argument('-l', "--LoadAddress", type=int, help='Specify payload load address', default =0x000800000)
+    parser.add_argument('-c', '--DscPath', type=str, required=True, help='Path to the DSC file')

```

Comment:
"required=True", please give the default value.

Reply:
done